### PR TITLE
Update icons and give cardGrid props

### PR DESCRIPTION
**Pull request by efrenaguilar95**
Updated icons for the character cards. Now, if there isn't an element, the weapon type will go to the top. I also made the CharacterGrid take a costumeInfo array because it should.

### CODE REVIEW

File: src/components/Card.tsx [hunk 1]
```diff
@@ -11,15 +11,14 @@ function Card(): JSX.Element {
 }
 
 function CharacterCard({ costume }: { costume: CostumeInfo }): JSX.Element {
+
   return (
-    <Link href={"/characters/" + costume.character + "/" + costume.name.en.replace(" ", "-")}>
+    <Link href={`/characters/${costume.character}/${costume.name.en.replace(" ", "-")}`}>
```
Comment (KeziahMoselle) [inline src/components/Card.tsx:16]: We should use slugify here

Comment (efrenaguilar95) [reply src/components/Card.tsx:16]: Ok what would that look like?
This?
`{`${slugify(`/characters/${costume.character}/${costume.name.en}}`)}`}`
Or this?
`{slugify(`/characters/${costume.character}/${costume.name.en}}`)}`

Comment (efrenaguilar95) [reply src/components/Card.tsx:16]: Markdown messed with formatting

image

Comment (KeziahMoselle) [reply src/components/Card.tsx:16]:
```jsx
<Link href={`/characters/${costume.character}/${slugify(costume.name.en, { lower: true, remove: /[*+~.()'"!:@]/g })}`}>
```

Comment (efrenaguilar95) [reply src/components/Card.tsx:16]: Now that I'm looking at it. I want to slugify characters as well. Should this be 2 separate slugifys? or one big one

Comment (KeziahMoselle) [reply src/components/Card.tsx:16]: Why characters ?

Comment (efrenaguilar95) [reply src/components/Card.tsx:16]: Well some might also have spaces, I can only think of one right now "The World Ender" but maybe more in the future?

Comment (KeziahMoselle) [reply src/components/Card.tsx:16]: ha you meant `costume.character` yeah

Comment (efrenaguilar95) [reply src/components/Card.tsx:16]: yeah, so two slugifys? or one big one. Not really sure what's better practice

Comment (KeziahMoselle) [reply src/components/Card.tsx:16]: Well yeah two call for slugify, I don't think slugify accept two slugify at once with the same parameters

Comment (efrenaguilar95) [reply src/components/Card.tsx:16]: well one would look like this..
```${slugify(`${costume.character}/${costume.name.en}`, { lower: true, remove: /[*+~.()'"!:@]/g })}`}>```

Comment (KeziahMoselle) [reply src/components/Card.tsx:16]: Oh yeah, might work? Idk lol

Comment (efrenaguilar95) [reply src/components/Card.tsx:16]: Ok it works, but also doesn't. Apparently the character urls are case sensitive. Going to update without `lower` for now, but maybe that's something we can do to character urls later on. (Make them case-insensitive, didn't know they would be in the first place :/)

Comment (KeziahMoselle) [reply src/components/Card.tsx:16]: I'll do it no worry

**Pull request merged**